batch-setup/make_rawr_tiles.py

```diff
@@ -31,29 +41,31 @@ def all_tiles_at(zoom):
 
 
 def missing_tiles(missing_bucket, rawr_bucket, date_prefix, region,
-                  key_format_type, config, zoom):
+                  key_format_type, config, zoom, tiles_coords_generator=None):
     from make_meta_tiles import MissingTileFinder
-
-    present = set()
-
-    finder = MissingTileFinder(
-        missing_bucket, rawr_bucket, date_prefix, date_prefix, region,
-        key_format_type, config, zoom)
-
-    with finder.present_tiles() as present_file:
-        with open(present_file) as fh:
-            for line in fh:
-                coord = deserialize_coord(line)
-                if coord.zoom == zoom:
-                    present.add(coord)
-
-    missing = set(all_tiles_at(zoom)) - set(present)
-    return missing
+    if bool(tiles_coords_generator):
+        return set([c for c in tiles_coords_generator.
+                   generate_tiles_coordinates([zoom])])
+    else:
+        present = set()
+        finder = MissingTileFinder(
```

Comment:
the type of change is for backward compatible, but I make it an argument without default value since our code base is not large and I change all the places.